frontend/src/components/Course/Course.tsx

```diff
@@ -82,8 +81,8 @@ const Course: React.FC<Props> = ({
           <MobileCourseProgress>
             <ProgressBar
               size={SIZE.medium}
-              text={`${progress}%`}
-              value={progress}
+              text={`${convertStatus(status)}%`}
```

Comment:
```suggestion
              text={convertStatus(status)}
```

it is unnecessary to use ${} in this case, because your function returns string